### ISSUE TITLE
80-column 1-pin driver

### ISSUE THREAD

**Issue by konimaru** [jacgoudsmit/L-Star#8]
For the record (since the OBEX managed to break the link), there is an 80 column version (@80MHz) or the 1-pin TV driver.

[80 column version](http://forums.parallax.com/discussion/comment/1265858/#Comment_1265858)
Also available [here](https://github.com/konimaru/waitvid.2048/tree/master/monochrome/Debug_1pinTV).

**Comment by jacgoudsmit** [comment]
Thanks!

So far there hasn't been a need for an 80-column 1-pin driver in L-Star but it might come in handy in the future.

80 columns is not really readable on most standard definition color TV screens though...

**Comment by konimaru** [comment]
Fair enough, anyway it's not getting lost (anymore). When I tested it I had a nice big 32" TV available. IIRC there is even an evidence picture in the forum thread.

I haven't looked in detail at your changes but did you move the font to hub RAM again?

**Comment by jacgoudsmit** [comment]
I saw the picture in the thread. It looks great! My 14" Trinitron will
definitely show it as well as your monitor, but I've used 27" CRT color
TV's that make a mess out of anything over 64 columns.

Monochrome monitors also have higher resolutions of course. I wouldn't be
surprised if my 5" black and white CRT TV would generate 80 colums without
blurring things too much.

For the OSI project, I had to move the font to the hub because it's 256
characters (not 128). The font would fill the entire cog. Unfortunately
that means that if the video clock is too high, the code can't retrieve the
video data fast enough. I think I did some calculations and wrote some
comments in the module about how far you can go.

===Jac
